data/README.md

```diff
@@ -4,109 +4,93 @@ A guide for managing a series of tubes.
 
 ## Best Practices
 
-* Avoid automatic retries for non-idempotent operations.
-* Use at least once delivery with idempotent operations.
-* Use at most once delivery with non-idempotent operations.
-* Use stream processing for data which is not guaranteed to fit into memory.
+- Avoid automatic retries for non-idempotent operations.
+- Use at least once delivery with idempotent operations.
+- Use at most once delivery with non-idempotent operations.
+- Use stream processing for data which is not guaranteed to fit into memory.
 
 ### Streaming
 
-* Avoid storing messages with different schemas in the same topic or queue.
-* Keep messages as small as possible.
-* Use as few distinct resources (CPU, Postgres, S3, HTTP requests) as possible
-  in each consumer.
-* Use constant memory to process streams when feasible.
-* When one message would perform more than one operation, instead have that
-  message append a new message for each operation.
-* When reading from the same queue with more than one consumer, ensure that it's
-  acceptable to process messages from that queue out of order.
-* When partitioning messages in a single topic, ensure that it's acceptable for
-  messages from different partitions to be processed out of order.
+- Avoid storing messages with different schemas in the same topic or queue.
+- Keep messages as small as possible.
+- Use as few distinct resources (CPU, Postgres, S3, HTTP requests) as possible in each consumer.
+- Use constant memory to process streams when feasible.
+- When one message would perform more than one operation, instead have that message append a new message for each
+  operation.
+- When reading from the same queue with more than one consumer, ensure that it's acceptable to process messages from
+  that queue out of order.
+- When partitioning messages in a single topic, ensure that it's acceptable for messages from different partitions to be
+  processed out of order.
 
 ## Glossary
 
-*Data Engineering*: the task of building infrastructure and process for
-ingesting, processing, and aggregating data so that it can be displayed to users
-or made available to data scientists.
+_Data Engineering_: the task of building infrastructure and process for ingesting, processing, and aggregating data so
```

Comment:
Wonder if we should make these `<dl>` lists.

Reply:
❤️ https://github.com/thoughtbot/guides/issues/619